actions/AuthAction.js

```diff
@@ -66,7 +68,8 @@ async function checkForProductContext(endpoint, clientId, org, token, productCon
 
 function asAuthAction(action) {
   return async (params) => {
-    const logger = Core.Logger('main', { level: params.LOG_LEVEL || 'info' });
+    logger = getLogger('AuthAction', params.AIO_LOG_LEVEL);
```

Comment:
Is AIO_LOG_LEVEL a request parameter or an environment variable? If it is an environment variable, then can we create an instance of a logger statically at the beginning of the file like this: const logger = Core.Logger('AuthAction')?

Reply:
@vtsaplin Didn't realize this was an option. I've made changes according to your suggestions.